cvmfs/cvmfs_config

```diff
@@ -245,6 +312,12 @@ cvmfs_chksetup() {
     fi
   done
 
+  # Check that CVMFS_USER is set
+  if [ x"$CVMFS_USER" = x"" ]; then
```

Comment:
I see.  Can you recover what went wrong?  Perhaps we can just move the check for required parameters to an earlier place.